### ISSUE TITLE
Opera Chrominuim Store

### ISSUE THREAD

**Issue by Havokdan** [pfn/passifox#359]
First, I do not speak English, I'm using google translator.
Secondly, good morning, thanks for the great software.
Third, I wonder if you can send a version for Opera Store, I'm using Chrome version in Opera, without any problems, it would be nice if other people who use Opera also had access.

**Comment by AleksueiR** [comment]
@Havokdan, you can use [Download Chrome Extensions](https://addons.opera.com/en/extensions/details/download-chrome-extension-9/) extension to install Chrome extensions in Opera.

**Comment by lebarondemerde** [comment]
@AleksueiR 

I can't make the extension work on Opera. I have installed it, it starts but I can't connect to anything. Clicking on "Connect" button does nothing.

**Comment by Havokdan** [comment]
Works for me

**Comment by lebarondemerde** [comment]
@Havokdan 

Yes, I had Opera configured to use a proxy I am testing, and I hed forgot about it. Disabling the proxy and everything went fine.

Thanks!
